spec/asar-spec.js

```diff
@@ -549,6 +550,60 @@ describe('asar package', function () {
       })
     })
 
+    describe('fs.exists', function () {
+      it('handles an existing file', function (done) {
+        var p = path.join(fixtures, 'asar', 'a.asar', 'file1')
+        // eslint-disable-next-line
+        fs.exists(p, function (exists) {
+          assert.equal(exists, true)
```

Comment:
Can we use `expect` from the `chair` package instead of `assert` please, it will help the work that @codebytere is doing to improve our test suite

Reply:
@MarshallOfSound I don't want that to be back-ported to 2.0. It's small amount of code compared to the rest of the tests.